core/pta/system.c

```diff
@@ -40,9 +40,9 @@ struct system_ctx {
 
 static unsigned int system_pnum;
 
-static TEE_Result system_rng_reseed(struct tee_ta_session *s __unused,
-				uint32_t param_types,
-				TEE_Param params[TEE_NUM_PARAMS])
+static TEE_Result system_rng_reseed(struct ts_session *s __unused,
+				    uint32_t param_types,
+				    TEE_Param params[TEE_NUM_PARAMS])
 {
 	size_t entropy_sz;
 	uint8_t *entropy_input;
```

Comment:
minor:could you add some init values here?

Reply:
OK